netutils/mac.py

```diff
@@ -86,7 +88,6 @@ def mac_to_int(mac):
     return int(mac_normalize(mac), 16)
 
 
-@_valid_mac
```

Comment:
Why remove this?

Reply:
I removed it to test the autodoc feature of Sphinx. I missed putting it back. Updated.